packages/demo/src/PostAddNext.re

```diff
@@ -23,6 +23,27 @@ module PostAddMutationConfig = [%graphql
 module PostAddMutation =
   ReasonApolloHooks.Mutation.Make(PostAddMutationConfig);
 
+module FieldString = {
+  [@react.component]
+  let make = (~field, ~label) => {
+    <PostAddForm.Field
+      field
+      render={({handleChange, error, value, validate}) => {
+        Js.log(label);
```

Comment:
log

Reply:
good catch, I actually had let to show how it only re-renders when changes are applied to it haha
I think I'll let it there with a comment